functions/Resolve-DbaNetworkName.ps1

```diff
@@ -173,12 +173,12 @@ Function Resolve-DbaNetworkName
 					Write-Verbose "$functionName - Getting computer information from $Computer via CIM (WSMan)"
 					if ($Credential)
 					{
-						$CIMsession = New-CimSession -ComputerName $Computer -ErrorAction SilentlyContinue -Credential $Credential
-						$conn = Get-CimInstance -Query "Select * FROM Win32_computersystem" -CimSession $CIMsession -ErrorAction SilentlyContinue
+						$CIMsession = New-CimSession -ComputerName $Computer -ErrorAction Stop -Credential $Credential
```

Comment:
Nope, these are done within a try block, so the catch will trigger.
I've added the same for the rest of the Cim-Calls, hushing everything. As a stop-gap for tomorrow it should suffice.